src/buildercore/terraform.py

```diff
@@ -56,6 +58,167 @@
 # see https://docs.fastly.com/guides/streaming-logs/changing-log-line-formats#available-message-formats
 FASTLY_LOG_LINE_PREFIX = 'blank' # no prefix
 
+# at the moment VCL snippets are unsupported, this can be worked
```

Comment:
all this new stuff should go into a `fastly` module, where the older Fastly code can be moved too later